frontend/app/services/GridService.scala

```diff
@@ -16,7 +16,11 @@ import play.api.libs.ws.WSRequest
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
+object GridService {
+  val cropQueryParam = "crop"
```

Comment:
Small style nit: should be capitalised http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods